docs/language/functions.md

```diff
@@ -42,6 +41,7 @@
   - [`nameof`](#nameof)
   - [`typeof`](#typeof)
   - [`typeunder`](#typeunder)
+  - [`quiet`](#quiet)
```

Comment:
Put this between nameof and typeof so the list remains alphabetical (and likewise with the target of this link).